libs/dataset-explorer/src/lib/ScatterUtils.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function getScatterColors(): string[] {
+  const colors = [];
+  for (let i = 0; i < 5; i++) {
+    colors.push("#6D8EF7", "#CA397E", "#F3B33E");
+  }
+  return colors;
+}
+
+export function getScatterSymbols(): string[] {
+  const symbols = [];
+  for (let i = 0; i < 3; i++) {
+    symbols.push("circle", "square", "diamond", "triangle", "triangle-down");
+  }
+  return symbols;
+}

```

Comment:
Could you add a simple unit test file for these two functions?

Reply:
added, thanks!
![image](https://user-images.githubusercontent.com/76190371/186545812-dc62215e-76c8-47a7-9a4e-67506b116755.png)